cedar-policy/CHANGELOG.md

```diff
@@ -40,7 +40,8 @@
   policy evaluation.
 - Fix a panic in `PolicySet::link` that could occur when the function was called
   with a policy id corresponding to a static policy.
-- Renamed `cedar_policy_core::est::EstToAstError` to `cedar_policy_core::est::FromJsonError`
+- Renamed `cedar_policy_core::est::EstToAstError` to `cedar_policy_core::est::FromJsonError`.
+- Renamed `cedar_policy_core::entities::JsonDeserializationError::ExtensionsError` to `cedar_policy_core::entities::JsonDeserializationError::FailedExtensionsFunctionLookup`.
```

Comment:
could also mention the rename of `ExtensionsError` itself, not just the variant of `JsonDeserializationError`.

Reply:
My rationale for not including this was that `ExtensionsError` is not exposed from `cedar-public`. Does that change anything?

Reply:
My bad, makes sense then.